docs/docs/30-administration/70-proxy.md

```diff
@@ -98,13 +98,40 @@ woodpeckeragent.example.com {
 Above configuration shows how to create reverse-proxies for web and agent communication. If your agent uses SSL do not forget to enable [WOODPECKER_GRPC_SECURE](./15-agent-config.md#woodpecker_grpc_secure).
 :::
 
+## Tunnelmole
+
+[Tunnelmole](https://github.com/robbie-cahill/tunnelmole-client) is an open source tunneling tool. Install Tunnelmole with one of the following methods depending on your system:
+
+- NPM: `npm install -g tunnelmole`
+- Linux: `curl -s https://tunnelmole.com/sh/install-linux.sh | sudo bash`
+- Mac: `curl -s https://tunnelmole.com/sh/install-mac.sh --output install-mac.sh && sudo bash install-mac.sh`
+- Windows: Install with NPM, or if you don't have NodeJS installed, download the `exe` file for Windows [here](https://tunnelmole.com/downloads/tmole.exe) and put it somewhere in your PATH.
+
+After installation, run the following command:
+
+```bash
+tmole 8000
+```
+
+It will start a tunnel and will give a response like this:
+
+```bash
+➜  ~ tmole 8000
+http://bvdo5f-ip-49-183-170-144.tunnelmole.net is forwarding to localhost:8000
+https://bvdo5f-ip-49-183-170-144.tunnelmole.net is forwarding to localhost:8000
+```
+
+Set `WOODPECKER_HOST` (for example in `docker-compose.yml`) to the Tunnelmole URL (`xxx.tunnelmole.net`) and start the server.
+
+````
+
 ## Ngrok
 
-After installing [ngrok](https://ngrok.com/), open a new console and run:
+[Ngrok](https://ngrok.com/) is a popular closed source tunnelling tool. After installing ngrok, open a new console and run the following command:
 
 ```bash
 ngrok http 8000
-```
+````
```

Comment:
Code block are closed by 3 `

Reply:
Looks like I missed that little typo, this is now fixed.